tests/test_matrix.py

```diff
@@ -14,6 +17,44 @@ def test_strMatrix():
     assert mat.__str__() == "[{1.0}+{x},{2.0};{3.0},{4.0}]"
 
 
+def test_traceMat():
+
+    mat = getTokens("[1, 2; \
+                      3, 4]")
+    Matrix.isSquare(mat)
```

Comment:
```suggestion
    mat.isSquare()
```
Same in line 30.
Everything else looks fine.